src/status_im/contacts/handlers.cljs

```diff
@@ -110,7 +110,7 @@
                              (filter (fn [[_ c]] (:global-command c)))
                              (map (fn [[id {:keys [global-command]}]]
                                     [(keyword id) (-> global-command
-                                                      (update :params vals)
+                                                      (update :params (comp vec vals))
                                                       (assoc :bot id
```

Comment:
This breaks over [here](https://github.com/status-im/status-react/blob/develop/src/status_im/chat/views/input/input.cljs#L137) because we can't `get-in` a seq. We could alternatively change `(get-in command [:command :params 0 :placeholder])` to `(-> command :command :params first :placeholder)` but personally I prefer to catch it at the source so I don't have to think about the differences between seqs and vectors.